test/unit/int_alma/scripts/helpers/almaRefundHelperTest.js

```diff
@@ -0,0 +1,22 @@
+// refundHelpers.js unit tests
+
+var assert = require('chai').assert;
+
+var order = {
+    getTotalGrossPrice: function () {
+        return 300;
+    },
+    orderNo: '00002222',
+    custom: {
+        almaPaymentId: 'payment_11uXoLs39IVLHMMQytcRzfcqiM6pdGqjuC'
+    }
+};
+
+var almaRefundHelpers = require('../../../../mocks/helpers/almaRefundHelpers');
+
+describe('almaRefundHelpers', function () {
+    it('check properties', function () {
+        var refundData = almaRefundHelpers.refundPaymentForOrder(order);
+        assert.equal(refundData, 'Refund is ok for the order n°' + order.orderNo);
+    });
+});
```

Comment:
Well done !!

Could you add some tests for :
- When the order is null
- a partial refund
- a partial refund with a negative amount
- a partial refund with an amount > to the total price

reach for me if you have questions/trouble :)